tests/php/Tasks/InvalidateAllSessionsTaskTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace SilverStripe\SessionManager\Tests\Services;
+
+use SilverStripe\Control\Middleware\ConfirmationMiddleware\Url;
+use SilverStripe\Control\Tests\HttpRequestMockBuilder;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\ORM\FieldType\DBDatetime;
+use SilverStripe\Security\AuthenticationHandler;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\RememberLoginHash;
+use SilverStripe\Security\Security;
+use SilverStripe\SessionManager\Extensions\RememberLoginHashExtension;
+use SilverStripe\SessionManager\Middleware\LoginSessionMiddleware;
+use SilverStripe\SessionManager\Models\LoginSession;
+use SilverStripe\SessionManager\Tasks\InvalidateAllSessionsTask;
+
+class InvalidateAllSessionsTaskTest extends SapphireTest
+{
+    use HttpRequestMockBuilder;
+
+    protected static $fixture_file = 'InvalidateAllSessionsTaskTest.yml';
+
+    protected static $required_extensions = [
+        RememberLoginHash::class => [
+            RememberLoginHashExtension::class,
+        ],
+    ];
+
+    public function testRemoveAllSessions()
+    {
+        // Ensure existing fixtured sessions are valid
+        DBDatetime::set_mock_now('2003-02-15 10:00:00');
+
+        // Log in as member 1 and make them active
+        $request = $this->buildRequestMock('dev/tasks/' . InvalidateAllSessionsTask::config()->get('segment'));
+        $request->method('getIP')->willReturn('192.168.0.1');
+        $member = $this->objFromFixture(Member::class, 'member1');
+        $handler = Injector::inst()->get(AuthenticationHandler::class);
+        $handler->login($member, true, $request);
+
+        // Ensure fixtured sessions are created and at least one login hash is generated
+        $this->assertNotSame(0, LoginSession::get()->count(), 'There are no LogginSessions');
```

Comment:
```suggestion
        $this->assertNotSame(0, LoginSession::get()->count(), 'There are no LoginSessions');
```

Reply:
Seems like we should be able to `assertSame(1, ` instead of `assertNotSame(0, ` ?

Reply:
I just care that it's not 0, not that it's explicitly 1. The test shouldn't fail if there are more sessions for whatever reason, because this assertion is just here to ensure there is something there which the task will soon clear out.